### ISSUE TITLE
[analysis] Support output-schema of a schema.core var-definition

### ISSUE THREAD

**Issue by ericdallo** [clj-kondo/clj-kondo#1268]
**Is your feature request related to a problem? Please describe.**
In clojure-lsp we would like to show the output schema of a var: https://github.com/clojure-lsp/clojure-lsp/issues/181

**Describe the solution you'd like**
Some way to retrieve the output-schema of a var-definition

**Describe alternatives you've considered**
I did a simple implementation here: https://github.com/clj-kondo/clj-kondo/pull/1267. We may need to think in some generic field for that though

**Additional context**
Maybe we could think in some common field name for all schema libraries like Mali, schema.core etc

**Comment by ericdallo** [comment]
Instead of my proposed `:output-schema-str`, we could think in something generic for all schema like libs, some extra field like `:schema` containing the output and input schemas maybe like:
```
{...
 :schema {:output 'some/Schema
          :input ['other/Schema 'another/Schema]}}
```
This way we would use that for probably all schema libraries, any thoughts @borkdude ?

**Comment by ericdallo** [comment]
@borkdude with #1211 merged, I think we can re-think this right? adding the schema to the :context, WDYT?

**Comment by borkdude** [comment]
Yes, with the context stuff in place, I think we can give this another try.

**Comment by ericdallo** [comment]
Still interested
